src/main/java/org/folio/orders/utils/HelperUtils.java

```diff
@@ -246,13 +245,13 @@ private static double accumulate(Double a, Double b) {
     return futures;
   }
 
-  public static CompletableFuture<JsonObject> operateOnSubObj(HttpMethod operation, String url,
-      HttpClientInterface httpClient, Context ctx, Map<String, String> okapiHeaders, Logger logger) {
-    return operateOnSubObj(operation, url, null, httpClient, ctx, okapiHeaders, logger);
+  public static CompletableFuture<JsonObject> operateOnObject(HttpMethod operation, String url,
+                                                              HttpClientInterface httpClient, Context ctx, Map<String, String> okapiHeaders, Logger logger) {
```

Comment:
sigh, we should really standardize on a code format.  I know I've shared one that I use with Kruthi.  Maybe we should put it up on the wiki or something

Reply:
There is [Eclipse Code Formatter](https://github.com/krasa/EclipseCodeFormatter#instructions) plugin available for IntelliJ IDEA which can apply Eclipse rules based on config file:
![image](https://user-images.githubusercontent.com/43410167/54411294-e428e180-46ff-11e9-8c66-31c9840dd34b.png)

Reply:
[FOLIO-1667](https://issues.folio.org/browse/FOLIO-1667) - there is [Eclipse style imports.xml](https://issues.folio.org/secure/attachment/16722/Eclipse%20style%20imports.xml) attached